rero_ils/modules/holdings/api_views.py

```diff
@@ -57,7 +57,7 @@ def decorated_view(*args, **kwargs):
             return jsonify({'status': 'error: {error}'.format(
                 error=error)}), 400
         except Exception as error:
-            raise(error)
+            raise Exception(error)
```

Comment:
@zannkukai does this break something for the `quick recieve` in the frontend?

Reply:
Nope, it shouldn't